ejb3/src/main/resources/schema/wildfly-ejb3_8_0.xsd

```diff
@@ -478,6 +478,13 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="legacy-compliant-principal-propagation" type="xs:boolean" use="optional" default="true">
+            <xs:annotation>
+                <xs:documentation>
+                    Option to switch between legacy compliant principal propagation or Elytron principal propagation.
```

Comment:
I suppose a simple explanation of the diference would be helpful.

Reply:
@OndrejKotek  fixed.